hw/ip/otbn/dv/otbnsim/sim/state.py

```diff
@@ -111,6 +114,10 @@ def __init__(self) -> None:
         # instruction. Make this a counter, decremented once per cycle. When we
         # get to zero, we set the flag.
         self._time_to_imem_invalidation = None  # type: Optional[int]
+        self.wiping_cnt = 0
```

Comment:
Please can you add a comment above this explaining how it works? (Something like "A count of how many cycles we've been in the WIPE state" maybe?)